onnxruntime/core/optimizer/graph_transformer_utils.cc

```diff
@@ -317,6 +318,12 @@ InlinedVector<std::unique_ptr<GraphTransformer>> GenerateTransformers(
 #else
       const bool avx2_precision_mode = false;
 #endif
+
+      if (session_options.config_options.GetConfigOrDefault(kOrtSessionOptionsDisableDoubleQDQRemover, "0") == "0") {
+        // DML doesn't support int16 QDQ, so we remove all pairs that cancel eachother to avoid falling back to the CPU
+        transformers.emplace_back(std::make_unique<Int16QDQPairsRemover>(dml_ep));
```

Comment:
I think L1 optimizations should be EP agnostic given partitioning hasn't occurred yet so we don't know which EP will be assigned the nodes.

Instead of using an L1 optimizer, can the DML EP create a ComputeCapability for a fused node that drops the DQ/Q? Should be relatively simple to do that.